pip/commands/wheel.py

```diff
@@ -172,7 +178,8 @@ def run(self, options, args):
                         global_options=options.global_options or [],
                         no_clean=options.no_clean,
                     )
-                    if not wb.build():
+                    wheels_built_successfully = wb.build(session=session)
```

Comment:
Likewise, good use of a variable to make it more clear what's going on here. I missed this in the original review, but moving things that have side effects out of if statements/while loops/etc tend to be a nice way to make code cleaner.